Sources/KlaviyoCore/Models/KlaviyoAPIError.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  KlaviyoAPIError.swift
+//
+//
+//  Created by Ajay Subramanya on 8/8/24.
+//
+
+import Foundation
+
+public enum KlaviyoAPIError: Error {
```

Comment:
This has to be public?

Reply:
yeah since it's used in `KlaviyoAPI` 
`public var send: (KlaviyoRequest, Int) async -> Result<Data, KlaviyoAPIError>`